.github/workflows/run_ert2_test_data_setups.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.6', '3.8', '3.10']
+        python-version: ['3.8', '3.10']
```

Comment:
Is there a reason only this test skips 3.9?

Reply:
Besides the build/test/deploy workflow which has to run all the Python version, it is only the `run-ert3-polynomial-local` workflow that runs Python 3.9. My plan was to address this in a separate PR, and at the same time consider removing some other runs of ert3 integration tests (eg. on macos) :slightly_smiling_face: